src/testUtils/mocks/polkadot.ts

```diff
@@ -622,6 +627,26 @@ export function createTxStub<
   return transactionMock;
 }
 
+/**
+ * @hidden
+ * Create and return an apollo query stub
+ *
+ * @param query - apollo document node
+ * @param returnValue
+ */
+export function createApolloQueryStub(query: DocumentNode, returnData: any): MockApolloClient {
+  const instance = mockInstanceContainer.apolloInstance;
+
+  instance.query = sinon
```

Comment:
What is the point of using 'mock-apollo-client' if you're just going to replace the query function with a sinon stub?